spec/lib/pwned_password_downloader_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+require 'pwned_password_downloader'
+require 'tempfile'
+
+RSpec.describe PwnedPasswordDownloader do
+  let(:destination) { Dir.mktmpdir('pwned_passwords') }
+  subject(:downloader) { PwnedPasswordDownloader.new(destination:) }
+
+  before do
+    stub_request(:get, URI.join(PwnedPasswordDownloader::RANGE_API_ROOT, '00000').to_s).to_return(
+      body: "0005AD76BD555C1D6D771DE417A4B87E4B4:10\r\n000A8DAE4228F821FB418F59826079BF368:4",
```

Comment:
I feel like the `\r\n` vs `\n` isn't particularly significant here since `each_line` takes care for us, so we could use heredoc syntax for a bit of readability?
```suggestion
      body: <<~BODY,
        0005AD76BD555C1D6D771DE417A4B87E4B4:10
        000A8DAE4228F821FB418F59826079BF368:4
      BODY
```

Reply:
> I feel like the `\r\n` vs `\n` isn't particularly significant here since `each_line` takes care for us, so we could use heredoc syntax for a bit of readability?

Updated in c56222175..